framework/src/modules/chain/transport.js

```diff
@@ -359,61 +356,64 @@ class Transport {
 			 * @todo Add @returns tag
 			 * @todo Add description of the function
 			 */
-			postBlock(query) {
+			async postBlock(query) {
 				if (!library.config.broadcasts.active) {
 					return library.logger.debug(
 						'Receiving blocks disabled by user through config.json'
 					);
 				}
 				query = query || {};
-				return library.schema.validate(
+
+				const valid = library.schema.validate(
```

Comment:
same as above. Maybe we can just import controller validator for now?

Reply:
This will be done as part of https://github.com/LiskHQ/lisk-sdk/issues/3278